reports/2023-project-director-elections-retrospective.md

```diff
@@ -0,0 +1,112 @@
+>    * Strive for tolerance and inclusivity by prioritizing selecting candidates that everyone can accept over selecting the first choice for only a subset of people.
+
+One challenge here was that the criteria was under-specified during the election process.
+This was a conscious choice in order to achieve the time-efficiency goal, so we chose to go with a minimal role description that we could refine over time rather than blocking on a perfect description.
+This mean the role description was actually a PR still under discussion for much of the process.
+
+This caused some challenges for nominees who had to decide whether to accept the nomination since the scope of the responsibilities was not entirely clear.
+
+There were also unclear eligibility criteria, particularly around employment limits.
+
+One strength here is that several of the nominees were outside of the set of well-known Rust project leaders, which suggests the nomination process was successful in helping to bring more people into project organization and leadership roles.
+
+### Conduct the Selection Process in a Time-efficient Manner
+
+To fully evaluate this, we need to know what we mean by time-efficient.
+
+One measure is the number of calendar days that it took to do the process.
+If we go by the blog posts that announced the beginning of the process and the end of the process, this took from August 30 until October 19, or 50 days.
+We needed to allow at least two weeks (14 days) for the nomination phase, and 10 days for the feedback phase, so the absolute minimum possible time would be 24 days.
+In practice, it's helpful to have some slack time between these phases, and we also needed time to coordinate the election meeting itself.
+Note that these 50 days do not include the time to design and plan for the process ahead of time.
+In the future, there should be less preplanning needed because we will not be designing the process from scratch.
+
+One reasons slack time in the schedule was helpful was that we neglected to plan time for nominees to consider whether to accept their nomination, and also time to confirm with the candidates who were selected.
+Going forward we should allow for this, because being willing to serve as a Project Director is a consequential decision that deserves adequate time to consider.
+
+Another measure of time efficiency is the time it took the people involved in the process to coordinate and do each of the steps.
+The role of Project Director Facilitator took a significant amount of time.
+It would be nice to reduce the time requirement, perhaps by subdividing the role.
+
+Relating to scheduling, we also need to consider scheduling constraints outside the election process itself.
+For example, candidate nominations closed on the last day of RustConf, and we were working on the blog post announcing the results of the election during the week of EuroRust.

```

Comment:
One point I should mention is that a conversation at RustConf actually led to the nomination I gave, and that candidate was later selected. Having the nomination deadline in the near future helped keep this possibility top of mind, though I agree that actually having it be *during* the conference is less than ideal.

Reply:
I wonder if it's worth using RustConf as an additional opportunity to socialize this process? Like maybe have announcements about the various election processes coming up and to consider who to nominate? We could try to schedule things so nominations close the week after a major conference to give more opportunity for this to happen.